packages/messenger/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+
+- Keep delegated handlers when unregistering actions ([#6395](https://github.com/MetaMask/core/pull/6395))
```

Comment:
Nit: Should this be changed instead? It was not a bug, it was done intentionally.